scania-digital-design-system/sdds
# feat(accordion-item): styling and refactoring

**Describe pull-request**  
Added a Host element to accordion and accordion-item to ease css styling. Refactored the accoridon-item styling and moved it to a separate file. Added a prop (mode-variant) to accordion. Added note of breaking changes in migration docs.

**Solving issue**  
Fixes: [AB#2820](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2820)

**How to test**  
1. Go to storybook link below
2. Check in Components -> Accordion
3. Check that the component behaves as it should.

**Screenshots**  
-

**Additional context**  
-
